schema-engine/sql-introspection-tests/tests/postgres/mod.rs

```diff
@@ -90,6 +90,7 @@ async fn pg_xml_indexes_are_skipped(api: &mut TestApi) -> TestResult {
     api.database().raw_cmd(&create_primary).await?;
 
     let dm = indoc! {r#"
+        /// This model contains an expression index which requires additional setup for migrations. Visit https://pris.ly/d/expression-indexes for more info.
```

Comment:
This links currently points to an issue tracking the feature. I think this is expected, but I wanted to double check!